cluster-autoscaler/core/scale_down.go

```diff
@@ -41,6 +42,14 @@ import (
 	"github.com/golang/glog"
 )
 
+// ScaleDownNonEmptyCandidatesCount is the maximum number of non empty nodes
+// considered at once as candidates for scale down.
+var ScaleDownNonEmptyCandidatesCount = flag.Int("scale-down-non-empty-candidates-count", 30,
```

Comment:
We try to keep flags in main.go and have them all in AutoscalingOptions.

Reply:
Done